kaska/parmap.py

```diff
@@ -247,11 +240,11 @@ def queue_getter():
 
 
 def _counter(items, tot=None):
-    for ii, item in enumerate(items):
+    for my_ii, item in enumerate(items):
```

Comment:
Change to `i`, unless there is a conflicting `i` in an outer scope, in which case rename to another of the standard index names {j,k,l,m,n}.

Reply:
Changed.